tailwind.config.js

```diff
@@ -8,6 +8,6 @@ export default /** @type {import('tailwindcss/tailwind-config').TailwindConfig}
   },
   plugins: [typography, daisyui],
   daisyui: {
-    themes: ['light', 'dark', 'cupcake', 'emerald', 'valentine', 'synthwave', 'halloween', 'aqua', 'dracula', 'winter']
+    themes: ["light", "dark", "cupcake", "bumblebee", "emerald", "corporate", "synthwave", "retro", "cyberpunk", "valentine", "halloween", "garden", "forest", "aqua", "lofi", "pastel", "fantasy", "wireframe", "black", "luxury", "dracula", "cmyk", "autumn", "business", "acid", "lemonade", "night", "coffee", "winter"],
```

Comment:
One thing to note is that adding too many themes can look complicated. Later I may re-pick some themes that fit Urara and provide a shortcut for those who want to use all of them.

Reply:
Sure!